config.go

```diff
@@ -45,6 +45,12 @@ func validateConfig(config *Config) error {
 	if config.InitialPacketSize > protocol.MaxPacketBufferSize {
 		config.InitialPacketSize = protocol.MaxPacketBufferSize
 	}
+	if config.MaxUDPPayloadSize > 0 && config.MaxUDPPayloadSize < protocol.MinInitialPacketSize {
+		config.MaxUDPPayloadSize = protocol.MinInitialPacketSize
```

Comment:
I was thinking should this be MinInitialPacketSize - 40 (IPv6 header) - 8 (UDP) header?

Reply:
wwasim838@gmail.com